cli/pcluster/config/validators.py

```diff
@@ -750,6 +750,10 @@ def s3_bucket_validator(param_key, param_value, pcluster_config):
             )
     except ClientError as client_error:
         _process_generic_s3_bucket_error(client_error, param_value, warnings, errors)
+    except ParamValidationError as validation_error:
+        errors.append(
+            "Error validating parameter '{0}'. Failed with exception: {1}".format(param_value, str(validation_error))
```

Comment:
I'd add the name of the parameter in the message.
Something like: Error validating parameter 'xx' with value 'xx'...

Reply:
Updated. The value of the parameter (the bucket name) is already present in the error message.